shared/eslint-config/restricted-syntax.js

```diff
@@ -10,6 +10,12 @@ module.exports = [
     message: "Don't refer to AbstractCrdt as a type. Use LiveNode instead.",
   },
 
+  {
+    selector: "TSTypeReference[typeName.name='WebSocket']",
```

Comment:
since this is a shared config, is there any chance this may leak out into a repo where we do want to be able to use the original websockets?